subsystem/cluster_test.go

```diff
@@ -1486,81 +1485,6 @@ var _ = Describe("cluster install", func() {
 					Expect(host.Bootstrap).Should(Equal(false))
 				}
 			})
-			It("[only_k8s]reset cluster and disable bootstrap", func() {
```

Comment:
Assuming we'll fix the reset one day, why not leave the existing unit tests for EnableAutoReset=True, and add new ones for EnableAutoReset=False?